src/plugins/intel_cpu/src/config.cpp

```diff
@@ -159,6 +159,14 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
                 IE_THROW() << "Wrong value for property key " << CPUConfigParams::KEY_CPU_DENORMALS_OPTIMIZATION
                 << ". Expected only YES/NO";
             }
+        } else if (key == PluginConfigInternalParams::KEY_SNIPPETS_SOFTMAX_ENABLE) {
```

Comment:
the same as for src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp. Leave a comment with a ticket reference maybe?